platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less

```diff
@@ -657,4 +657,11 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       }
     }
   }
+
+  .pinnedActivity {
+    border: 1px solid @greyColorLighten1Default;
+    background-color: white;
+    border-radius: 3px;
+    color: @greyColorLighten1Default;
```

Comment:
Use @greyColorLighten1 color variable instead of @greyColorLighten1Default.

Reply:
Done!